api/src/main/java/marquez/db/OpenLineageDao.java

```diff
@@ -283,12 +283,22 @@ default void upsertLineageDataset(
       source =
           sourceDao.upsert(
               UUID.randomUUID(),
-              getSourceType(ds),
+              getSourceType(ds).getValue(),
+              now,
               now,
               ds.getFacets().getDataSource().getName(),
-              ds.getFacets().getDataSource().getUri());
+              ds.getFacets().getDataSource().getUri(),
+              ds.getFacets().getDescription());
```

Comment:
This is a bug in the open lineage implementation! There shouldn't be a description field here. That must have accidentally been carried over from porting the openapi spec.